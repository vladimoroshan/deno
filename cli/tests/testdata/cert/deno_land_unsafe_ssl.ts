const r = await fetch("https://localhost:5545/cert/cafile_info.ts");
console.log(r.status);
