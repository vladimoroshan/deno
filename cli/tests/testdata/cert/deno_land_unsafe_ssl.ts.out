DANGER: TLS certificate validation is disabled for: localhost
200
